github-actions/create-pr-for-changes/lib/main.ts

```diff
@@ -129,6 +129,7 @@ async function main(): Promise<void> {
     const {
       data: {items: supersededPrs},
     } = await git.github.search.issuesAndPullRequests({
+      advanced_search: 'true',
```

Comment:
Yeah, when I read the commit description, I remembered that we already had logic for this in place. Is this just a fix because it didn't work?

Reply:
Yes